features/exploredash/build.gradle

```diff
@@ -14,7 +14,7 @@ android {
         targetSdkVersion 30
         versionCode 1
         versionName "1.0"
-
+        vectorDrawables.useSupportLibrary = true
```

Comment:
why do we need this?

Reply:
We're updating everything on 23, I think my physical-search PR has this change. You can bump it up and I think we should dump support lib